packages/kit/src/views/Discovery/pages/Dashboard/Welcome/Welcome.tsx

```
@@ -54,6 +55,8 @@ function DappSideDisplay({
 }
 
 export function Welcome({ banner }: { banner: React.ReactNode }) {
+  const media = useMedia();
+  
```

ESLint

[error] Delete `··`

(prettier/prettier)